frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-items.tsx

```diff
@@ -233,12 +287,14 @@ export const OperatorHubTileView: React.FC<OperatorHubTileViewProps> = (props) =
   const [detailsItem, setDetailsItem] = React.useState(null);
   const [showDetails, setShowDetails] = React.useState(false);
 
+  const filteredItems = filterByArchAndOS(props.items);
```

Comment:
nit: this should probably be memoized so that it doesn't get recalculated on every render.

Reply:
Nice, that would be a good optimization. I'll make a note for a follow.